### ISSUE TITLE
Users hit "Association error : one Entity is unable to find another entity" during Avni-client sync

### ISSUE THREAD

**Issue by himeshr** [avniproject/avni-client#683]
Of late, we have been receiving complaints of internal team members as well as external users hitting "Association error : one Entity is unable to find another entity" during Avni-client sync. Refer attached screenshot for example.

One of the users has also raised a ticket for the same: https://avni.freshdesk.com/helpdesk/tickets/2177

![WhatsApp Image 2022-09-22 at 9.40.06 AM.jpeg](https://images.zenhubusercontent.com/6261483184a61f6d60f9114a/0d883be1-a97e-4a55-abf1-1ba0bfefa133)

Recommended workaround to unblock users is as follows: 
Update the last_modified_date_time for the missing entity in DB and thereafter another attempt at sync should resolve the issue.

**Comment by himeshr** [comment]
We further investigated the issue and found that there are checklists created for "Phulwari" Program for MCH beneficiaries.
Since the user doesnot have access to "Phulwari" Program, we are unable to sync the checklist to the app.
We need to delete or re-associate the checklists so that its not mapped for MCH beneficiaries.

**Comment by himeshr** [comment]
We further investigated the issue and found that there are checklists created for "Phulwari" Program for MCH beneficiaries.
Since the user doesnot have access to "Phulwari" Program, we are unable to sync the checklist to the app.
We need to delete or re-associate the checklists so that its not mapped for MCH beneficiaries.

**Comment by vinayvenu** [comment]
@himeshr is this the same as https://app.zenhub.com/workspaces/avni-5cf8e458bf08585333fd64ac/issues/avniproject/avni-client/688?

**Comment by himeshr** [comment]
Duplicate of #688. Please refer [#688](https://app.zenhub.com/workspaces/avni-5cf8e458bf08585333fd64ac/issues/avniproject/avni-client/688), for more information on issue resolution.

**Comment by himeshr** [comment]
Duplicate of #688.
